packages/daemon/src/reader-ref.js

```diff
@@ -60,9 +60,6 @@ export const makeIteratorRef = iterable => {
       }
       return harden({ done: true, value: undefined });
     },
-    [Symbol.asyncIterator]() {
-      return this;
-    },
```

Comment:
@kriskowal is this a non-test use? Do we need to preserve it?

Reply:
I think this is vestigial and safe to remove. I don’t see any `E(ref)[Symbol.asyncIterator]()`.